src/scanner.c

```diff
@@ -1710,12 +1724,21 @@ static bool inner_scan(void *payload, TSLexer *lexer, const bool *valid_symbols)
             break;
 
         case '%':
+            lex_advance(lexer);
+
+            // valid_symbols[MACRO_CONTROL_END] &&
```

Comment:
Does this need to be commented out?

Reply:
Yeah it was a sort of todo / reminder to come back and check this. Having it as part of the if stopped it from working 🙃